doc/devdocs/development/new-powertoy.md

```diff
@@ -0,0 +1,258 @@
+
+> **Gotcha:** PowerToys caches `.nuget` artifacts aggressively. Use `git clean -xfd` when builds behave unexpectedly.
+---
+## 7. Installer and Packaging (WiX)
+
+### Add Your Module to Installer
+
+1. Install [`WixToolset.Heat`](https://www.nuget.org/packages/WixToolset.Heat/) for Wix5 via nuget
+2. Inside `installer\PowerrToysInstallerVNext` add a new file for your module: `Module.wxs`
+3. Inside of this file you will need copy the format from another module (ie: Light Switch) and replace the strings and GUID values.
+4. The key part will be `<!--ModuleNameFiles_Component_Def-->` which is a placeholder for code that will be generated by `generateFileComponents.ps1`.
+5. Inside `Product.wxs` add a line item in the `<Feature Id="CoreFeature" ... >` section. It will look like a list of ` <ComponentGroupRef Id="ModuleComponentGroup" />` items.
+6. Inside `generateFileComponents.ps1` you will need to add an entry to the bottom for your new module. It will follow the following format. `-fileListName <Module>Files` will match the string you set in `Module.wxs`, `<ModuleServiceName>` will match the name of your exe.
+```bash
+# Module Name
+Generate-FileList -fileDepsJson "" -fileListName <Module>Files -wxsFilePath $PSScriptRoot\<Module>.wxs -depsPath "$PSScriptRoot..\..\..\$platform\Release\<ModuleServiceName>"
+Generate-FileComponents -fileListName "<Module>Files" -wxsFilePath $PSScriptRoot\<Module>.wxs -regroot $registryroot
+```
+---
+## 8. Testing and Validation
+
+### UI Tests
+* Place under `/modules/<YourModule>/Tests`
+* Create new [WinUI Unit Test App](https://learn.microsoft.com/en-us/windows/apps/winui/winui3/testing/create-winui-unit-test-project)
+* Write unit tests following format from previous modules (ie: Light Switch). This can be to test your standalone UI (if you're a module like Color Picker) or to verify that the Settings UI in the PowerToys app is controlling your service.
+
+### Manual Validation
+* Enable/disable in PowerToys Settings
+* Check initialization in logs
+* Confirm icons, tooltips, and OOBE page appear correctly
+
+### Pro Tips:
+1. Validate wake/sleep and elevation states. Background modules often fail silently after resume if event handles aren’t recreated.
+2. Use Windows Sandbox to simulate clean install environments
+3. To simulate a "new user" you can delete the PowerToys folder from `%LOCALAPPDATA%\Microsoft`
+---
+## 9. Out-of-Box Experience (OOBE) page
+The OOBE page is a custom settings page that gives the user at a glance information about each module. This Window opens first before the Settings application for new users and after updates. Create `OOBE<ModuleName>.xaml` at ` src\settings-ui\Settings.UI\SettingsXAML\OOBE\Views`. You will also need to add your module name to the enum at `src\settings-ui\Settings.UI\OOBE\Enums\PowerToysModules.cs`
+--
```

Comment:
A few things that come to mind that are missing (in Step 10):

- Module Icon
- Module Image
- OOBE Image
- Docs

I'd maybe call these out as something to consider but we can handle those for the sake of consistency.